src/functionalTest/resources/application-functional.yaml

```diff
@@ -7,7 +7,7 @@ generated.user.email.pattern: ${GENERATED_USER_EMAIL_PATTERN:freg-test-user-%s@f
 
 test.user.password: ${TEST_USER_PASSWORD:Hmcts123}
```

Comment:
Please check both secrets idam.auth.tokenAuth, idam.auth.clientSecret can be removed, as those are real but i think we are not using